data/auxiliary/gather/ldap_query/ldap_queries_default.yaml

```diff
@@ -10,19 +10,42 @@ queries:
       - cACertificateDN
       - dNSHostname
       - certificateTemplates
-  - action: ENUM_ADCS_CERT_TEMPLATES
-    description: 'Enumerate ADCS certificate templates.'
+      - objectGUID
+      - caCertificate
```

Comment:
Description for this should probably be updated as technically speaking its Enterprise CAs not standard CAs since its using ActiveDirectory, but maybe I'm being nit picky here?